docs/getting_started_jax.md

```diff
@@ -0,0 +1,252 @@
+# Shardy guide for JAX users
+
+Shardy is a new propagation system being introduced into the XLA stack, and below we want to introduce any JAX users to:
+
+1. What has changed in JAX
+2. Why Shardy?
+3. Future plans
+
+This is meant for JAX users who use `jax.jit` for running training/inference models across more than 1 GPU or TPU (batch parallelism, megatron, ZeRO, etc). They would be using things like `PartitionSpec`s and `NamedSharding`s.
+
+## 1. What has changed in JAX?
+
+### State of JAX before: GSPMD
+
+Prior to Shardy, JAX users who partitioned their models across models across multiple devices used [GSPMD](http://go/arxiv/2105.04663) behind the scenes.
+
+GSPMD is the propagation+partitioning system that lives in the middle of the XLA pipeline. It operates on HLO - the IR that comes after StableHLO (the program you get after running `jax.jit.lower`).
+
+JAX doesn't run GSPMD directly, but encodes instructions into the StableHLO IR.
+
+But before we go any further, let's introduce our working example.
+
+Note that this example should be executed in a TPU environment with 8 devices.
```

Comment:
This is content from a colab, and ideally we can use a "Open in colab" link as well, but I'm not sure how to do this for the documentation engine used for Shardy.

Reply:
Markdown:

```
[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openxla/stablehlo/blob/main/docs/tutorials/jax-export.ipynb)
```

Renders as

[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openxla/stablehlo/blob/main/docs/tutorials/jax-export.ipynb)

Reply:
The rendering engine can also render `.ipynb` colab's as documentation:

https://openxla.org/stablehlo/tutorials/jax-export

is an auto-generated rendering of: https://github.com/openxla/stablehlo/blob/main/docs/tutorials/jax-export.ipynb

Reply:
Removed md file, added ipynb.